src/desktop/apps/authentication/helpers.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from "reaction/Components/Authentication/Types"
 import { data as sd } from "sharify"
 import { pickBy, identity } from "lodash"
+import * as qs from "query-string"
```

Comment:
Total sidenote, looking into our `yarn.lock`(https://github.com/artsy/force/blob/c9e1edf14335d065961a112c2867e1828c136084/yarn.lock#L11841-L11891) file, we seem to have at least 2 different query string libraries actively being used in code.

querystring: https://github.com/artsy/force/blob/c9e1edf14335d065961a112c2867e1828c136084/src/desktop/apps/articles/components/AuthWrapper.tsx#L3
query-string: https://github.com/artsy/force/blob/c9e1edf14335d065961a112c2867e1828c136084/src/desktop/apps/armory_week/index.js#L7

We also seem to explicitly include `qs` in `package.json`: https://github.com/artsy/force/blob/c9e1edf14335d065961a112c2867e1828c136084/package.json#L228 , a 3rd library (doesn't seem to be actively used however).

Strangeness!

Reply:
Yeah, I noticed something along these lines as well. What’s your recommendation here on action to take? I’d prefer to defer that cleanup considering that this PR is pretty meaty as is.

Reply:
I also agree that deferring that cleanup sounds good.

Reply:
For sure! I think those libraries are tiny (string-parsing), compared to things like moment, was just looking out for some duplication in that spirit.